quiz.js

```diff
@@ -0,0 +1,99 @@
+//******************************JavaScript Quiz******************************\\
+
+//DOM ELEMENT VARIABLE DEFINITIONS
+var mainContent = document.getElementById('mainContent');
+var heightInput = document.getElementById('heightInput');
+var characterInput = document.getElementById('characterInput');
+var submitButton = document.getElementById('submitButton');
+var clearButton = document.getElementById('clearButton');
+
+
+//OBJECT AND ARRAY INIT FOR TREE VALUES
+var treeSpecs = {};
+var treeArray = [];
+
+
+//CLEARS HTML MAIN CONTENT DIV AND INPUTS
+var clear = function () {
+	mainContent.innerHTML = "";
+	heightInput.value = "";
+	characterInput.value = "";
+};
+
+
+//PRINTS FORMATTED ARRAY TO DOM AND CONSOLE
+var arrayPrinter = function(array, elem) {
+	var arrayString = array.join(" ").toString();
```

Comment:
If you look at the example output, there are no spaces between the characters making up the tree.